README.md

```diff
@@ -1,61 +1,63 @@
-
-- The `setDebugMessageLevel(int _debugLevel)` method is used to set a level of granularity in information output. Every debug message comes with a level which goes from 0 to 4. A higher level means more verbosity. Debug messages with a level higher than `_debugLevel` will not be shown. The lowest level has higher importance and is usually used for errors, which are always printed. Passing -1 as a parameter will disable logging entirely, **errors will also be ignored**.
-
-### ArduinoIoTCloud
-
-- The `begin(ConnectionManager *connection = ArduinoIoTPreferredConnection, String brokerAddress = "mqtts-sa.iot.arduino.cc")` method is used to initialize a connection to the Arduino IoT Cloud through MQTT. Accepts an instance of `ConnectionManager` and returns 0 in case of failure or 1 if object is successfully instantiated.
-
-- `begin(Client& net, String brokerAddress = "mqtts-sa.iot.arduino.cc")` if connection is managed manually and not via a `ConnectionManager` a reference to a `Client` object can be used.
-
-- The `connect()` method is used to connect to the MQTT broker.
-
-- `disconnect()` closes the connection to the MQTT Client.
-
-- The `update()` method can be called periodically in the loop of the `.ino` file. If a `ConnectionHandler` is implemented it checks network connections. It also makes sure that a connection with the MQTT broker is active and tries to reconnect otherwise. During `update()` data from the Cloud is retrieved and changed values are posted to the proper MQTT topic.
-
-- `connected()` simply returns the current status of the MQTT connection.
-
-- `reconnect(Client&)` cleans up the existing MQTT connection, creates a new one and initializes it by calling `connect()`
-
-- `setThingId(String const thing_id)` sets the **THING_ID** to properly configure the MQTT topic.
-
-- `getThingId()` returns the **THING_ID**.
-
-- `connectionCheck()` invokes the `check()` method from a **ConnectionHandler** if it is implemented. Mainly it implements a state machine and is responsible for the connection to Arduino IoT Cloud.
-
-- `getIoTStatus()` returns the Cloud connection status. The different states are defined in an `enum`
-
+- **Properties**: Qualities defining the characteristics of a system. A _Property_ can be defined as *read-only* (`READ`) to indicate that Arduino IoT Cloud can read the data, but cannot change the value of such _Property_. On the other end, it may be designated to be **read-and-write** (`READWRITE`), allowing Arduino IoT Cloud to remotely change the property’s value and trigger an event notification on the device. It is also possible to mark properties as *write-only* (`WRITE`) which means the cloud can write to this property but not read it's value (this limits data transmission for properties which are used to trigger events only).
```

Comment:
```suggestion
- **Properties**: Qualities defining the characteristics of a system. A _Property_ can be defined as *read-only* (`READ`) to indicate that Arduino IoT Cloud can read the data, but cannot change the value of such _Property_. On the other end, it may be designated to be **read-and-write** (`READWRITE`), allowing Arduino IoT Cloud to remotely change the property’s value and trigger an event notification on the device. It is also possible to mark properties as *write-only* (`WRITE`) which means the cloud can write to this property but not read its value (this limits data transmission for properties which are used to trigger events only).
```
it's -> its